home/src/main/resources/rdf/i18n/en_CA/interface-i18n/firsttime/vitro_UiLabel.ttl

```diff
@@ -6270,6 +6334,30 @@ uil-data:no_individual_associated_with_id.Vitro
         uil:hasKey      "no_individual_associated_with_id" ;
         uil:hasPackage  "Vitro-languages" .
 
+uil-data:password_reset_admin_notification_email_subject.Vitro
```

Comment:
Translations for keys below are provided only in en_CA and en_US . 
Were they lost during merge conflict resolution?
password_reset_admin_notification_email_subject
password_reset_admin_notification_email_plain_text
password_reset_admin_notification_email_html

Reply:
No, this was intentional as we wanted to discuss email content during one of the meetings, and only after aggreeing on it, we would add translations. But we forgot about it :smile: